cypress/e2e/pages/location.cy.ts

```diff
@@ -142,15 +142,20 @@ describe('Location', () => {
     });
 
     context('when back button is pressed in browser', () => {
+      before(() => {
+        cy.visit('/locations/STO-002');
+      });
       it('should display the updated list of awaiting labwares', () => {
-        cy.go('back');
         cy.findByText('STAN-2111').should('exist');
         cy.findByText('STAN-4111').should('exist');
         cy.findByText('STAN-3111').should('not.exist');
       });
     });
 
     context('when refreshing the page', () => {
+      before(() => {
+        cy.reload();
```

Comment:
Does this need to reload in both the before() and subsequently in the `it('should...)` block?